config/releases.exs

```diff
@@ -283,3 +283,38 @@ config :service_acquire, Acquire.Application,
   ]
 
 config :service_acquire, Acquire.Db.Presto, presto: Keyword.put(presto_db, :user, "acquire")
+
+# SERVICE DEFINE
+config :service_define, DefineWeb.Endpoint,
+  http: [:inet6, port: String.to_integer(System.get_env("DEFINE_PORT") || "4005")],
+  secret_key_base: secret_key_base,
+  live_view: [
+    signing_salt: secret_key_base,
+  ],
+  render_errors: [view: DefineWeb.ErrorView, accepts: ~w(json)],
+  pubsub: [name: Define.PubSub, adapter: Phoenix.PubSub.PG2],
+  server: true,
+  check_origin: false
+
+config :define_broadcast, Broadcast.Application,
```

Comment:
What is this?

Reply:
Gottum!
(I just got burnt by a very similar typo)

Reply:
Fixed this